tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml

```diff
@@ -210,10 +210,10 @@
                                     <RowDefinition/>
                                     <RowDefinition/>
                                 </Grid.RowDefinitions>
-                                <TextBlock Text="Error logs" Grid.Row="0" Grid.Column="0" Margin="0, 5, 0, 0" Foreground="{StaticResource TextFillColorPrimary}"/>
+                                <TextBlock Text="Error logs" Grid.Row="0" Grid.Column="0" Margin="0, 5, 0, 0"/>

```

Comment:
[Not part of this PR, but noticing it now] @dhoehna or @AmelBawa-msft, whoever owns this page -- is the text "Error logs" here and "View log" below a problem? Should this be localized?

Reply:
I would say it should be localizable, unless that's a debug feature only? Adding @dhoehna for context.

Reply:
Yes, it should be localizable.  That was my mistake.

Reply:
Opened #1670